components/enterprise-contract/.tekton/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+resources:
+- serviceaccount.yaml
+- pipelines-as-code.yaml
+- pvc.yaml
+
```

Comment:
Update https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build/shared-resources/shared-resources-components.yaml to allow namespace used by this service to access the secrets.